src/haxeLanguageServer/features/haxe/codeAction/OrganizeImportsFeature.hx

```diff
@@ -187,9 +187,15 @@ class OrganizeImportsFeature {
 
 	static function organizeImportGroups(doc:HxTextDocument, context:Context, importGroups:Map<Int, ImportGroup>):Array<TextEdit> {
 		var edits:Array<TextEdit> = [];
+		var last = importGroups.get(-1);
+		importGroups.remove(-1);
+
 		for (group in importGroups) {
 			edits = edits.concat(organizeImportGroup(doc, context, group));
 		}
+
+		if (last != null)
+			edits = edits.concat(organizeImportGroup(doc, context, last));
```

Comment:
we should make the tests not rely on a particular order of edits, but rather test that all required edits are present, regardless of order.

but this seems to work and it doesn't add any additional load on everyday operation, so I guess it'll do for now.

Reply:
I'm not too familiar with these code actions so I took the safe route of making it behave exactly like before.